torch_tvm/compiler.cpp

```diff
@@ -222,29 +233,40 @@ void TVMCompiler::run(Stack& stack) {
     value_to_ivalue[value_input] = inputs[i];
   }
 
-  CompleteArgumentSpec spec{false, ArrayRef<IValue>(inputs)};
-
-  if (cache_.find(spec) == cache_.end()) {
+  if (!cache_ || (cache_ && (*cache_).invalid)) {
```

Comment:
if `cache_` is an optional, can we get rid of the `invalid` attribute somehow?  it's pretty confusing